BlinkID/www/blinkIdScanner.js

```diff
@@ -387,6 +387,28 @@ BlinkID.prototype.RecognitionMode = Object.freeze(
     }
 );
 
+/**
+ * Defines possible document card rotation positions
+ */
+ BlinkID.prototype.CardRotation = Object.freeze(
+    {
+    /** Zero degrees */
+    Zero: 0,
+
+    /** Clockwise 90 degrees */
+    Clockwise90: 1,
+
+    /** Counter clockwise 90 degrees */
+    CounterClockwise90: 2,
+
+    /** Upside down */
+    UpsideDown: 3,
+
+    /** Card rotation was not performed */
```

Comment:
This comment should be something like "Determining card rotation was not performed"